### ISSUE TITLE
[Breadcrumbs]: wcag color issues

### ISSUE THREAD

**Issue by kristiina-richards** [TEHIK-EE/tedi-design-system#79]
### Description of the problem

1. The contrast between link and text in breadcrumbs is 1.16:1. it should be 3:1
**Requirement:** [link](https://www.w3.org/WAI/WCAG22/Understanding/use-of-color.html)

**Comment by kristiina-richards** [comment]
Leaving to be fixed when moved to TEDI components
